app/views/external_pledges/create.js.erb

```diff
@@ -1 +1 @@
-// rerender #existing-external-pledges
\ No newline at end of file
+$('#external_pledge_amount').val('');
```

Comment:
I expect that you'll want to massage this just a bit. When you hit submit, on a successful thing, you're looking to do the following:

* clear the form (in this case, both the amount and the source -- probably you can just copy this line and change the id and be good to go)
* re-rack the external pledges table partial so that the new external pledge shows up.

In notes, which you've done a great job modeling this after, we do it like this: 

```
$('#notes_log').empty().append('<%= escape_javascript(render partial: "notes/notes_table", locals: { notes: @notes } ) %>');
```

That is -- we empty the div with existing external pledges in it, and then replace its contents with the partial, which is now accounting for the new entry.